datalad/interface/run.py

```diff
@@ -611,7 +611,7 @@ def run_command(cmd, dataset=None, inputs=None, outputs=None, expand=None,
                 ofh.write(assure_bytes(msg))
             lgr.info("The command had a non-zero exit code. "
                      "If this is expected, you can save the changes with "
-                     "'datalad save -r -F %s .'",
+                     "'datalad add -d . -r -F %s .'",
```

Comment:
Now thinking about it, this wouldn't be a correct instruction in case of `--explicit` run... Not sure what would be a sensible cross platform and concise way though, besides ` add` (may be adding it as a feature for all datalad API functions getting a list of files) getting an option of taking a file with filenames to add, and then is preparing that file alongside with the message file to be given here.
I will leave it for you to decide to move that into a new issue/PR and here for explicit mode may be just say that it might add too much or listing all outputs?

Reply:
> this wouldn't be a correct instruction in case of `--explicit` run

When I've thought about it in the past, I've decided it's not worth worrying about `--explicit` here.  If you feeling strongly, please open a PR tweaking the message as you see fit.